models/comment.js

```diff
@@ -284,11 +284,11 @@ CommentSchema.statics.pushStatus = (id, status, assigned_by = null) => Comment.u
  * @param {String} action the new action to the comment
  * @return {Promise}
  */
-CommentSchema.statics.addAction = (item_id, user_id, action_type) => Action.insertUserAction({
+CommentSchema.statics.addAction = (item_id, user_id, action) => Action.insertUserAction({
   item_id,
   item_type: 'comment',
   user_id,
-  action_type
+  action
```

Comment:
This style (hiding fields inside this new `action` value) makes this difficult to read.